onnxruntime/core/providers/cpu/quantization/qlinearconv.cc

```diff
@@ -577,9 +629,13 @@ Status QLinearConv<ActType>::Compute(OpKernelContext* context) const {
     padding_data.resize(static_cast<size_t>(C), X_zero_point_value);
   }
 
-  int32_t thread_count = ComputeThreadCount(output_image_size, group_output_channels, kernel_dim);
   concurrency::ThreadPool* thread_pool = context->GetOperatorThreadPool();
+#if defined(_M_ARM64) || defined(__aarch64__)
+  int32_t thread_count = (output_image_size + 3) / 4;
```

Comment:
>4 [](http://example.com/codeflow?start=51&length=1)

Could you please add the comment to clarify the reason to choose 4 here (to align with the row count of kernel)?

And we need to change the name thread_count. It is actually task/work count. We can do it in a separate PR though.